editor.planx.uk/src/@planx/components/PlanningConstraints/Public.tsx

```diff
@@ -69,32 +69,22 @@ function Component(props: Props) {
   const urlSearchParams = new URLSearchParams(window.location.search);
   const params = Object.fromEntries(urlSearchParams.entries());
 
-  // Get the coordinates of the site boundary drawing if they exist, fallback on x & y if file was uploaded
-  // Coords should match Esri's "rings" type https://developers.arcgis.com/javascript/3/jsapi/polygon-amd.html#rings
-  const coordinates: number[][][] = siteBoundary?.geometry?.coordinates || [];
-
-  // Get the WKT representation of the site boundary drawing or address point to pass to Digital Land, when applicable
+  // Get the WKT representation of the site boundary drawing or address point to pass to Planning Data
   const wktPoint = `POINT(${longitude} ${latitude})`;
   const wktPolygon: string | undefined =
     siteBoundary && stringify(siteBoundary);
 
-  const digitalLandParams: Record<string, string> = {
+  const planningDataParams: Record<string, string> = {
     geom: wktPolygon || wktPoint,
     ...params,
   };
-  const customGisParams: Record<string, string> = {
-    x: x?.toString(),
-    y: y?.toString(),
-    siteBoundary: JSON.stringify(coordinates),
-    version: "1",
-  };
 
   // Fetch planning constraints data for a given local authority
   const root = `${import.meta.env.VITE_APP_API_URL}/gis/${teamSlug}?`;
   const teamGisEndpoint: string =
     root +
     new URLSearchParams(
-      hasPlanningData ? digitalLandParams : customGisParams,
+      hasPlanningData ? planningDataParams : undefined,
```

Comment:
Are we relying on this failing because `geom` will be undefined when the request is made?

 If `!hasPlanningData` should be guard against making the request full stop?

Reply:
Already need to revisit these request conditions in #3968 (eg only make "roads" request _if_ the editor has specified they want to check roads and they have a USRN) - so going to take these comments forward into that PR is ok!